api/ws/rpc/connect.js

```diff
@@ -45,20 +45,22 @@ const connectSteps = {
 	addConnectionOptions: peer => {
 		const systemHeaders = System.getHeaders();
 		const queryParams = {};
-
-		if (systemHeaders.version != null) {
+		if (systemHeaders.version !== null && systemHeaders.version !== undefined) {
 			queryParams.version = systemHeaders.version;
 		}
-		if (systemHeaders.wsPort != null) {
+		if (systemHeaders.wsPort !== null && systemHeaders.wsPort !== undefined) {
```

Comment:
same here and wherever we have the check for `null` and `undefined` explicitly in if in below code.

Reply:
Your suggestion @ManuGowda returns false for the following: `"", false, 0, undefined, null`, whereas the code below only for `undefined, null`.